Essentials/src/com/earth2me/essentials/signs/EssentialsSign.java

```diff
@@ -375,7 +376,8 @@ protected final ItemStack getItemMeta(final ItemStack item, final String meta, f
         return stack;
     }
 
-    protected final BigDecimal getMoney(final String line, final IEssentials ess) throws SignException {
+    protected final BigDecimal getMoney(String line, final IEssentials ess) throws SignException {
+        line = ChatColor.stripColor(line);
```

Comment:
Why is this change here?

Reply:
I color the lines if it's updated from worth. So you can see which signs use the values from worth.yml. If I don't strip the color here we won't be able to get the price value since there could be non-number (color) characters on this line.